.github/ISSUE_TEMPLATE/new_collection_inclusion.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Do we need this type of issue here? Up until now, we tracked inclusion requests as discussions on the https://github.com/ansible-collections/ansible-inclusion/ repo.

Reply:
@tadeboro I created those based on https://github.com/ansible-community/community-topics/issues/2 not to discuss here and only as a topic to discuss in the meeting but it can go in the "General Topic" issue template as well. Will remove it.